packages/css/src/accordion/accordion.scss

```diff
@@ -0,0 +1,63 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+@import "../../utils/breakpoint";
+
+.amsterdam-accordion__header {
+  display: flex;
+  margin-block: 0;
+  margin-inline: 0;
+}
+
+.amsterdam-accordion__button {
+  background-color: transparent;
+  border: 0;
+  color: var(--amsterdam-accordion-button-color);
+  cursor: pointer;
+  display: flex;
+  font-family: var(--amsterdam-accordion-button-font-family);
+  font-size: var(--amsterdam-accordion-button-font-size-narrow);
+  font-weight: var(--amsterdam-accordion-button-font-weight);
+  justify-content: space-between;
+  line-height: var(--amsterdam-accordion-button-line-height);
+  padding-block: 0.75rem;
+  padding-inline: 1rem;
+  width: 100%;
+
+  @media screen and (width > $amsterdam-breakpoint) {
+    font-size: var(--amsterdam-accordion-button-font-size-wide);
+  }
+
+  &:focus {
+    outline-offset: var(--amsterdam-accordion-button-focus-outline-offset);
+  }
+
+  &:focus-visible,
+  &:hover {
+    box-shadow: var(--amsterdam-accordion-button-hover-box-shadow);
+  }
+
+  svg {
+    transform: rotate(0deg);
+    transition: transform 0.3s ease;
+
+    @media (prefers-reduced-motion) {
+      transition: none;
+    }
+  }
+}
+
+.amsterdam-accordion__button[aria-expanded="true"] svg {
+  transform: rotate(180deg);
+}
+
+.amsterdam-accordion__panel {
+  display: none;
+  padding-block: 1rem;
+  padding-inline: 1rem;
+}
+
+.amsterdam-accordion__panel--expanded {
```

Comment:
A nice prefers reduced motion query for an animated icon but nog for the expanding of the content. Seems like a missed opportunity.

Reply:
I really dislike animating the expanding of content for accordions, it just means you have to wait longer before you see the text.